cookbook/attributes/default.rb

```diff
@@ -25,6 +25,12 @@
 default['redisio']['package_install'] =
   Chef::VersionConstraint.new('>= 14.04').include?(node['platform_version'])
 
+# Set to true if redis-server should be installed
+default['redis']['local'] = true
+
+# Set to true if mysql should be installed
+default['database']['local'] = true
+
```

Comment:
These should get switched by the run_list instead of attributes. I separated the redis and mysql installation resources into a recipe so that we could just exclude it and not install either.